index.html

```diff
@@ -0,0 +1,213 @@
+                <div class="w-72 h-60 transition duration-500 ease-in-out hover:bg-gray-700 transform hover:-translate-y-1 hover:scale-110 bg-gray-300 rounded-md shadow-2xl hover:text-white">
+                    <div class= " mt-3  flex flex-row items-center  text-center gap-x-1 "> 
+                       
+                      <i class=" mx-3 fas fa-tools fa-lg"></i>
+                         <span class="mx-3 text-3xl font-light hover:text-white hover:bg-gray-700">Tools</span>
+                   </div>
+                   <hr class="border-1  mt-2 border-yellow-700"/>
+                   <div class="mx-12 mt-3.5 font-light" >
+                        <p class="mb-3.5 text-lg  font-light transform hover: motion-reduce:transform-none ">VS Code</p>
+                        <p class="  mb-3.5 text-lg  font-light transform hover: motion-reduce:transform-none">Git </p>
+                        <p class="  mb-3.5 text-lg  font-light transform hover: motion-reduce:transform-none">NPM </p>
+                        <p class="  mb-3.5 text-lg  font-light transform hover: motion-reduce:transform-none">Jira </p>
+                         
+                   </div>
+                </div>
+                <!--Education -->
+
+                <div class="w-72 h-60 transition duration-500 ease-in-out hover:bg-gray-700 transform hover:-translate-y-1 hover:scale-110 bg-gray-300 rounded-md shadow-2xl hover:text-white">
+                  <div class= " mt-3  flex flex-row items-center  text-center gap-x-1 "> 
+                     
+                    <i class=" mx-3 fas fa-tools fa-lg"></i>
+                       <span class="mx-3 text-3xl font-light hover:text-white hover:bg-gray-700">Education</span>
+                 </div>
+                 <hr class="border-1 mt-2 border-yellow-700"/>
+                 <div class="mx-12 mt-3.5 font-light" >
+                      <p class="mb-3.5 text-lg  font-light transform hover: motion-reduce:transform-none ">IT Management</p>
+                      <p class="  mb-3.5 text-lg  font-light transform hover: motion-reduce:transform-none">IS Software Engineer </p>
+                      <p class="  mb-3.5 text-lg  font-light transform hover: motion-reduce:transform-none">SFC </p>
+                      <p class="  mb-3.5 text-lg  font-light transform hover: motion-reduce:transform-none">Freecodecamp </p>
+                       
+                 </div>
+              </div>
+
+                
+            </div>
+           
+      </section>
+      <section class="container mx-12 mt-12  mb-12">
+            <div class=" text-gray-700 mb-12 mt-12 text-center uppercase font-bold text-3xl "> work</div>
+            <div class="flex flex-row">
+      <!---first project-->
+        <div class=" group  bg-yellow-200 relative w-1/4  m-5  overflow-hidden">
+          <img src="./Images/2.jpg" alt="Avatar" class="object-cover w-full h-full opacity-50 transition duration-500   group-hover:opacity-100 " />
```

Comment:
It's also a good idea to give more descriptive names to all your files, including images. This way it's easier for someone that is looking at the codebase to understand what a file is, and where it will be used.